### ISSUE TITLE
Apalache integration on Windows doesn't work well

### ISSUE THREAD

**Issue by bugarela** [informalsystems/quint#1414]
We currently download apalache and spawn a bash script to run the server. On windows, we can't simply run a bash script.

I tried adding a `.bat` file to the apalache release (locally, as a test), but I'm having issues getting node to run that `bat` in a child process. I need to dig a bit further to make this work. If I manage to make this work, we'd also have to update Apalache to include the bat file on it's release artifact.

Right now, windows users have to run the Apalache server themselves (i.e. by using the jar), and then quint can connect to it just fine

**Comment by konnov** [comment]
I guess, you could check the platform and spawn the bat file here:

https://github.com/informalsystems/quint/blob/ae270804d1e0d30b461c9b932ef40fd3b87d3463/quint/src/apalache.ts#L360

However, if the bash file would work in WSL, why not recommend Windows users to use WSL?

**Comment by bugarela** [comment]
Yes, I tried that yesterday and it works (see https://github.com/informalsystems/quint/compare/main...gabriela/fix-windows-apalache). That's why I finally opened the PR on Apalache with the `.bat`.

I guess WSL is not completely trivial to set up. Some of my students struggled with that last semester. Experienced developers should have no issue with this, but making Windows support more seamless is going to help the group who can't figure things out by themselves.
